examples/twittermap/script/dockerRunMySqlDB.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -
+#===============================================================================
+#
+#          FILE: dockerRunMySqlDB.sh
+#
+#         USAGE: ./dockerRunMySqlDB.sh
+#
+#   DESCRIPTION: Run MySQL docker containers
+#
+#       OPTIONS:
+#  REQUIREMENTS: ---
+#          BUGS: ---
+#         NOTES: ---
+#        AUTHOR: Haohan Zhang (), hh1680651@gmail.com
+#  ORGANIZATION: ics.uci.edu
+#       CREATED: 08/14/2017 23:24:01 PM PDT
+#      REVISION:  ---
+#===============================================================================
+
+set -o nounset                      # Treat unset variables as an error
+
+echo "Starting docker-machine ..."
+docker-machine create --driver virtualbox default
+docker-machine start
+eval $(docker-machine env)
+
+HOST="$(echo -e "$(docker-machine ip)" | sed -e 's/^[[:space:]]*//')"
+PORT=3308
+
+echo "Editing config files ..."
```

Comment:
why do we need to edit the cloudberry config file in the script? 
I think the script should only take care of the setup the MySQL part. The config file is meant to be changed by a human.